src/rotest/management/base_resource.py

```diff
@@ -237,7 +236,6 @@ def enable_debug(self):
         debug(self.initialize, ignore_exceptions=[KeyboardInterrupt, BdbQuit])
```

Comment:
we can call this method,
it should be in abstract_test.py -> request resources I think

Reply:
Are you talking about the `reset` method or the `initialize`?
I couldn't find any occurrence of `reset` except on `ut_models.py`, and I deleted it as well

Reply:
Currently the resources aren't debugged when using -D
Because we never call the enable_debug method of the resources after locking them